gulpfile.js

```diff
@@ -84,46 +84,35 @@ gulp.task('server', () => {
 
 gulp.task('test:app', () =>
   gulp.src(paths.testSpecs + 'app_spec.js', {read: false})
-  .pipe(mocha({reporter: 'spec'}))
+  .pipe(mocha({reporter: 'spec', exit: true}))
   // https://github.com/sindresorhus/gulp-mocha#test-suite-not-exiting
```

Comment:
Is this comment still relevant, or should it be removed?

Reply:
It was still relevant as were using something from that part of the docs. However I've taken it out as we were essentially just linking to the docs for the library itself which seems redundant.